akka-http-caching/src/main/java/akka/http/caching/javadsl/Cache.java

```diff
@@ -1,6 +1,7 @@
-/**
- * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com/>
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
AFAIU the year range was meant to keep track of when the file was created. For example, `Cache` was created on 2017, so this year should be 2017-2018. Is this something legal? Or just a good thing to have?

Reply:
A nice to have AFAIR. We can use the automatic template hm

Reply:
> AFAIU the year range was meant to keep track of when the file was created. For example, Cache was created on 2017, so this year should be 2017-2018. Is this something legal? Or just a good thing to have?

That's one way to use/interpret them but it seems that Akka uses the convention to use the same headers everywhere (e.g. https://github.com/akka/akka/commit/6c7085252a4bb8e5761af3a2c08e7f006e657c8a where everything was changed to 2017). It would make sense to keep it persistent between Akka  sub-projects. When it comes to history, there's always the commit history as the ultimate truth.